core-processor/src/ext.rs

```diff
@@ -503,6 +505,24 @@ impl EnvExt for Ext {
 
         self.charge_sending_fee(delay)?;
 
+        if delay != 0 {
+            // Take delay and get cost of block
+            // Calculate reserve like  wait_cost * (delay + reserve_for)
+            // Mikita todo
```

Comment:
Resolve TODO

Reply:
It's useless, I just forget to delete TODO